integrations/optimizely/lib/index.js

```diff
@@ -118,9 +118,12 @@ Optimizely.prototype.track = function(track) {
 
   push(payload);
 
-  var optimizelyClientInstance = window.optimizelyClientInstance;
+  var optimizelyOptions = track.options('Optimizely');
```

Comment:
Does this always resolve to an object? If it ever resolves to `null` or `undefined`, then `optimizelyOptions.FullStackClientInstance` will throw a `TypeError`.